ui/apps/auth-app/src/components/connect/choose-provider.tsx

```diff
@@ -67,7 +67,7 @@ const ChooseProvider: React.FC<ChooseProviderProps> = props => {
           {subtitleLabel}
         </Text>
 
-        <Box direction="column" pad="small" style={{ width: '100%' }}>
+        <Box direction="column" style={{ width: '100%' }}>
```

Comment:
you can also use the width prop directly here instead of in the style object

Reply:
thanks for the fix!